README.md

```diff
@@ -438,6 +438,27 @@ Creating a search key that will only search over the body field.
 {'source_engines': ['source-engine-1', 'source-engine-2'], 'type': 'meta', 'name': 'my-meta-engine'}
 ```
 
+### Search the API logs
+
+```python
+>>> client.search_api_logs('my-meta-engine', {
```

Comment:
Could you change to this `get_api_logs` just to be consistent with other clients:
https://github.com/elastic/app-search-php/blob/master/resources/api/api-spec.yml#L838
https://github.com/elastic/app-search-php